app/controllers/order_items_controller.rb

```diff
@@ -0,0 +1,27 @@
+class OrderItemsController < ApplicationController
+  before_action :find_order_item, only: [:destroy, :qty_decrease, :qty_increase]
+
+  def find_order_item
+    @order_item = OrderItem.find(params[:id])
+  end
+
+  def destroy
+    @order_item.destroy
+    @order_item.save
+    redirect_to order_path
+  end
+
+  def qty_decrease
+    @order_item.quantity -= 1
+    @order_item.save
+
+    redirect_to :back rescue redirect_to order_items_decrease_path
```

Comment:
This redirect would be a GET request, but the `order_items_decrease` only works for POST requests. I'm not sure this is where you would want to redirect the user if `:back` doesn't exist.